api/src/importers/alc/inspections-utils.js

```diff
@@ -0,0 +1,82 @@
+const BaseRecordUtils = require('./base-record-utils');
+const CsvUtils = require('./utils/csv-utils');
+const MiscConstants = require('../../utils/constants/misc');
+
+/**
+ * ALC csv Inspections record handler.
+ *
+ * @class Inspections
+ */
+class Inspections extends BaseRecordUtils {
+  /**
+   * Creates an instance of Inspections.
+   *
+   * @param {*} auth_payload user information for auditing
+   * @param {*} recordType an item from record-type-enum.js -> RECORD_TYPE
+   * @param {*} csvRow an object containing the values from a single csv row.
+   * @memberof Inspections
+   */
+  constructor(auth_payload, recordType, csvRow) {
+    super(auth_payload, recordType, csvRow);
+  }
+
+  /**
+   * Convert the csv row object into the object expected by the API record post/put controllers.
+   *
+   * @returns an inspection object matching the format expected by the API record post/put controllers.
+   * @memberof Inspections
+   */
+  transformRecord(csvRow) {
+    if (!csvRow) {
+      throw Error('transformRecord - required csvRow must be non-null.');
+    }
+
+    const inspection = { ...super.transformRecord(csvRow) };
+
+    inspection['_sourceRefStringId'] = Number(csvRow['record id']) || '';
```

Comment:
Shouldn't this be a string based on the name?

Reply:
Yup I'll set it to a string to match the field name